src/main/java/org/jboss/xavier/analytics/pojo/output/workload/summary/RecommendedTargetsIMSModel.java

```diff
@@ -115,6 +133,7 @@ public String toString() {
                 ", rhv='" + rhv +
                 ", rhel=" + rhel +
                 ", osp=" + osp +
+                ", cnv=" + cnv +
```

Comment:
here we can expect that if cnv is null , this will print "null" , is that what we want ?